updater/spec/dependabot/updater/operations/group_update_all_versions_spec.rb

```diff
@@ -289,4 +289,72 @@
       group_update_all.perform
     end
   end
+
+  context "when the snapshot is updating vendored dependencies", :vcr do
+    let(:job) do
+      Dependabot::Job.new_update_job(
+        job_id: "1558782000",
+        job_definition: job_definition_with_fetched_files,
+        repo_contents_path: create_temporary_content_directory(fixture: "bundler_vendored")
+      )
+    end
+
+    let(:job_definition) do
+      job_definition_fixture("bundler/version_updates/group_update_all_with_vendoring")
+    end
+
+    let(:dependency_files) do
+      # Let's use the already up-to-date files
+      original_bundler_files(fixture: "bundler_vendored")
+    end
+
+    before do
+      stub_rubygems_calls
```

Comment:
I think you can update [#stub_rubygems_calls](https://github.com/dependabot/dependabot-core/blob/f602c6372fb127a4470481ed3702be82530b53d8/updater/spec/support/dummy_pkg_helpers.rb#L8) to also stub out the vendored gem. The VCR returns a `401 repository not found` right now, so you probably don't need it?

Reply:
Yeah, that's a bug that the smoke tests are tripping into, I need to update the test to use a 'real' gem instead.

Reply:
Smoke test output for reference here: https://github.com/dependabot/smoke-tests/pull/84

Reply:
I've investigated this a bit more and realised it's not what I thought - I'm going to leave the stubs for this alone for now since we're tolerating it via VCR and it is kind of inconsequential to the test for now.